src/snowflake/sqlalchemy/snowdialect.py

```diff
@@ -230,20 +239,19 @@ def create_connect_args(self, url):
         )
         return ([], opts)
 
-    def has_table(self, connection, table_name, schema=None):
+    def has_table(self, connection, table_name, schema=None, **kw):
         """
         Checks if the table exists
         """
         return self._has_object(connection, "TABLE", table_name, schema)
 
-    def has_sequence(self, connection, sequence_name, schema=None):
+    def has_sequence(self, connection, sequence_name, schema=None, **kw):
```

Comment:
These functions get passed a new argument `info_cache`. Not certain what it is for, but kwargs makes this change backwards-compatible and we can keep the old implementation for now.